perllib/FixMyStreet/App/Controller/Reports.pm

```diff
@@ -365,13 +365,13 @@ sub body_find : Private {
     my ($self, $c, $q_body) = @_;
 
     # We must now have a string to check
-    my @bodies = $c->model('DB::Body')->search( { name => { -like => "$q_body%" } } )->all;
+    my @bodies = $c->model('DB::Body')->active->search( { name => { -like => "$q_body%" } } )->all;
```

Comment:
Sadly, this deliberately checks deleted bodies, so that URLs for old now-deleted councils like https://www.fixmystreet.com/reports/Penwith still work. I guess it should try active ones first, and only then try deleted ones if it doesn't find one.